packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -49,6 +49,7 @@ const defaultConfigs = {
     lookback: 7200,
     invertPrice: true,
     minTimeBetweenUpdates: 60,
+    priceFeedDecimals: 8,
```

Comment:
For `USDBTC` contracts that are currently deployed we are using an 8 decimal price feed.

Reply:
We should do the same for BCHNBTC right?